doc/sphinx/source/references.bib

```diff
@@ -134,3 +134,28 @@ @article{zhang2011verification
   year={2011},
   pages={807--827}
 }
+
+@book{bower2010applied,
+  title={Applied mechanics of solids},
+  author={Bower, Allan F},
+  year={2010},
+  publisher={CRC press},
+  isbn={978-1-4398-0247-2}
+}
+
+
+@book{Holzapfel2002,
+  title     = "Nonlinear Solid Mechanics: A Continuum Approach for Engineering
+               Science",
+  author    = "Holzapfel, Gerhard A",
+  abstract  = "Introduction to Vectors and Tensors. Kinematics. The Concept of
+               Stress. Balance Principles. Some Aspects of Objectivity.
+               Hyperelastic Materials. Thermodynamics of Materials. Variational
+               Principles. References. Index.",
+  journal   = "Meccanica",
+  publisher = "Springer Nature",
+  volume    =  37,
+  number    = "4/5",
+  pages     = "489--490",
+  year      =  2002
```

Comment:
```suggestion
  year      =  2000,
  isbn      = {978-0-471-82319-3}
```
Looks like the year is 2000 according to
https://www.wiley.com/en-us/Nonlinear+Solid+Mechanics%3A+A+Continuum+Approach+for+Engineering-p-9780471823193

Reply:
yeah, looks like my citation manager pulled the wrong thing. I am fixing it now.